specification/healthdataaiservices/HealthDataAIServices.DeidServices/routes.tsp

```diff
@@ -31,23 +33,45 @@ alias DeidentifyOperations = ResourceOperations<
 #suppress "@azure-tools/typespec-azure-core/use-standard-names" "Existing name"
 @summary("Create a de-identification job.")
 @pollingOperation(getJob)
-op createJob is DeidentifyOperations.LongRunningResourceCreateOrReplace<DeidentificationJob>;
+op deidentifyDocuments is DeidentifyOperations.LongRunningResourceCreateOrReplace<DeidentificationJob>;
 
 @summary("Get a de-identification job.")
 op getJob is DeidentifyOperations.ResourceRead<DeidentificationJob>;
 
 @summary("List de-identification jobs.")
+@access(Access.internal, "csharp,python")
```

Comment:
I believe these TCGC decorators should be in client.tsp (`@@access(HealthDataAIServices.DeidServices.listJobs, ...)`)